pontus_sensors/CMakeLists.txt

```diff
@@ -27,39 +27,16 @@ include_directories(
   $(rclcpp_INCLUDE_DIRS)
   ${OpenCV_INCLUDE_DIRS}
   colormaps
```

Comment:
I think you can remove this line too

Reply:
Are you refering to the colormaps one, or all 3?